src/js/events/events.ts

```diff
@@ -162,6 +162,11 @@ export const AD_ERROR = 'adError';
 */
 export const AD_IMPRESSION = 'adImpression';
 
+/**
+ * Event triggered on instream adapter when vast media is loaded onto the video tag.
+ */
+export const AD_MEDIA_LOADED = 'adMediaLoaded';
```

Comment:
Instream doesn't need the work 'ad' prefixed on it's events. Only ad events in the public API need that to distinguish between content and ad events.

This can simply be changed to `'mediaLoaded`.